scripts-dev/run-localdev-setup.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# This script assumes to be run from the root of the following directory layout:
+#  - complement: a checkout of the complement repo, with homeserver built
+#  - trafficlight: a checkout of this repo
+#  - matrix-react-sdk: a checkout of matrix-react-sdk, where the trafficlight adapter can be found.
+
+# Additionally, an environmental variable `ELEMENT_WEB_LOCATION` should be set
+# with the location of the `element-web` repository where the dev server can be run from.
+
+systemctl list-units --type=service | grep -q docker
```

Comment:
```suggestion
echo "Docker Version: "
docker info -f '{{.ServerVersion}}'
```

Tests that the docker client can connect to a server and gets the version, doesn't depend on systemd being used to start it.

Reply:
This will still exit 0 if a connection is found, exit 1 otherwise